### ISSUE TITLE
[Windows 10 x32]: Agent in unhealthy state when installed with default policy having System and Endpoint Security.

### ISSUE THREAD

**Issue by amolnater-qasource** [elastic/beats#25108]
**Kibana version:** 8.0 Snapshot Kibana cloud environment

**Host OS and Browser version:** Windows 10 x32, All

**Build Details:**
```
  Artifact link used: https://snapshots.elastic.co/8.0.0-ceb812fb/downloads/beats/elastic-agent/elastic-agent-8.0.0-SNAPSHOT-windows-x86.zip
  Build: 41953
  Commit: cb3c4e3a212255a9e9b8c89e784e0e452b661233
```

**Preconditions:**
1. 8.0 Snapshot Kibana cloud environment should be available.

**Steps to reproduce:**
1. Login to Kibana environment.
2. Install agent with default policy having System and Endpoint Security  integrations.
3. Navigate to agent "Logs" tab and observe no logs and agent in "Unhealthy" state.

**Note:**
-  We attempted to restart the agent to bring it back to "Healthy" state, however unable to resolve it.

**Expected Result:**
Agent must be installed on Windows 10 x32 and should remain in "Healthy" state.

**[debug] level Agent logs:**
[debug level agent logs.zip](https://github.com/elastic/beats/files/6316474/debug.level.agent.logs.zip)

**Screenshots:**
![Win32](https://user-images.githubusercontent.com/77374876/114840922-eb49de80-9df4-11eb-96c1-6ac52bddaff0.jpg)

**Comment by amolnater-qasource** [comment]
@muskangulati-qasource Please review.

**Comment by muskangulati-qasource** [comment]
Reviewed & assigned to @EricDavisX

**Comment by EricDavisX** [comment]
I don't see a statement about Fleet Server and so I assume it was run with Kibana connection - we aren't supporting that moving forward.  we can re-test with Fleet Server usage in 7.13 or 8.0 and report back if it isn't working.  @muskangulati-qasource Amol has steps for running Fleet Server, I know they are changing recently

**Comment by amolnater-qasource** [comment]
Hi @EricDavisX 
Thanks for the feedback, we will revalidate this whenever the complete Fleet Server changes will be available on 8.0/7.13 Snapshot build. 

For now we are closing this with respect to the new changes for Kibana 7.13. 

Thanks 
QAS
